AspNetCore.Diagnostics.HealthChecks.sln

```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 15
-VisualStudioVersion = 15.0.28010.2036
+# Visual Studio Version 16
+VisualStudioVersion = 16.0.28407.52

```

Comment:
Are you using VS 2019? this is not a release version, I prefer to mantain 15 version